src/pages/result/index.tsx

```diff
@@ -2,23 +2,29 @@ import GetImage from "../../components/GetImage";
 import { DefaultButton } from "../../components/ui/Button";
 import styles from "./index.module.css";
 
-function Result() {
+type ResultProps = {
+	score: number;
+};
+
+const Result = ({ score }: ResultProps) => {
 	const images = [
 		"/drink/bottle0.webp",
 		"/drink/bottle1.webp",
 		"/drink/bottle2.webp",
 		"/drink/bottle3.webp",
 	];
 
+	const image = images[score] || images[0];
```

Comment:
```suggestion
	const image = `/drink/bottle${score}.webp`
```
このほうが綺麗じゃね？
あと、`images[score]`がnullになる場合ってある？
(それよりはscoreがimagesのlengthを越えてないかのチェックとかの方があった方がいい気がする)

Reply:
確かに,scoreがimagesのlengthを越えてないかのチェックの方がいいですね、、書き換えました！